sdk/src/network/public/fetcher/fetcher-impl.ts

```diff
@@ -151,4 +152,8 @@ export class WhirlpoolAccountFetcher implements WhirlpoolAccountFetcherInterface
   ): Promise<ReadonlyMap<string, PositionBundleData | null>> {
     return this.fetcher.getAccounts(addresses, ParsablePositionBundle, opts);
   }
+  populatePools(pools: [Address, WhirlpoolData][]): void {
+    const poolMap = new Map(pools.map(([address, pool]) => [AddressUtil.toString(address), pool]));
+    this.fetcher.populateAccounts(poolMap, ParsableWhirlpool, Date.now());
+  }
```

Comment:
Is there a specific reason to implement this feature for WhirlpoolData only ? 🤔

I think this feature is for advanced users who control the cache, and I don't think we need to move in the direction of adding populateX for each data type.

Can we generalize this feature like as follows ? 
```
  populateAccounts<T extends WhirlpoolSupportedTypes>(
    accounts: ReadonlyMap<string, T | null>,
    parser: ParsableEntity<T>,
    now: number
  ): void {
    this.fetcher.populateAccounts(accounts, parser, now);
  }
```